test/e2e/run_test.go

```diff
@@ -609,6 +609,13 @@ USER bin`, BB)
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToString()).To(Equal("111"))
+
+		currentOOMScoreAdj, err := ioutil.ReadFile("/proc/self/oom_score_adj")
```

Comment:
Could we turn this into a system test?  Just to be extra sure we catch any potential regression in RHEL (gating).